modules/@apostrophecms/piece-type/index.js

```diff
@@ -607,6 +600,9 @@ module.exports = {
       // within a single lock and without redundant network operations. This greatly
       // improves the performance of saving all changes to a document at once after
       // accumulating a number of changes in patch form on the front end.
+      //
+      // If `input._publish` launders to a boolean and the type is subject to draft/publish
```

Comment:
I got confused here. It seems to suggest that it'll pass if it's simply registered as a boolean. It also has to be true.

Reply:
Thanks for the opportunity to say "launders to a truthy boolean" with a straight face.

Reply:
"Truthy" is one of programming's greatest gifts to the world.